ops/framework.py

```diff
@@ -138,8 +138,11 @@ def __repr__(self):
     def defer(self):
         """Defer the event to the future.
 
-        After being deferred, this event will be re-emited the next time there's
-        any other event for the charm.
+        After being deferred, this event will be re-emitted the next time there's
+        any other event for the charm. However the current event handler will
+        continue execution, in the absence of an explicit return statement. Further
+        when the event is re-emitted, the corresponding event handler will being
```

Comment:
s/being/begin/

Reply:
> s/being/begin/

@jameinel @chipaca replaced my docstring with a commit suggestion above. I like his suggestions and want to accept it. But he wanted your second opinion on his suggested edit above . So this spelling error can be ignored if we accept his much better doc string.